packages/sources/src/addSourceWizard/hardcodedComponents/aws/access_key.js

```diff
@@ -8,14 +8,14 @@ Create an access key in your AWS user account and enter the details below.
     </Text>
     <Text component={ TextVariants.p }>
 For sufficient access and security, Red Hat recommends using
-    the Power User Identity and Access Management (IAM) policy for your AWS user account.&nbsp;
+    the Power user identity and access management (IAM) policy for your AWS user account.&nbsp;
         <Popover
             aria-label="Help text"
             position="bottom"
             bodyContent={
                 <React.Fragment>
                     <Text component={ TextVariants.p }>
-                        The Power User IAM policy allows the user full access to API functionality and AWS services for user administration.
+                        The Power user policy allows the user full access to API functionality and AWS services for user administration.
```

Comment:
Should not this be `The power user`?

Reply:
Not sure, I am following the JIRA description.

Reply:
Ok.